medusa/subtitle_providers/legendastv.py

```diff
@@ -246,7 +249,7 @@ def search_titles(self, title):
                     if match:
                         title['season'] = int(match.group('season'))
                     else:
-                        logger.warning('No season detected for title %d', title_id)
+                        logger.debug('No season detected for title id: %d and name: %s', title_id, name)
```

Comment:
If this PR were without the log message changes, then we could easily PR upstream. 
This warning shouldn't change to debug